apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx

```diff
@@ -0,0 +1,413 @@
+      console.log('res: ', res)
+    })
+  }, [account, parentAccount, transaction]);
+
+  const onChange = useCallback(index => {
+      if (index != null) {
+        setTransaction(bridge.updateTransaction(transaction, { index }));
+      }
+    },
+    [setTransaction, transaction, bridge],
+  );
+
+  const onContinue = () => {
+    console.log(`transaction is: ${JSON.stringify(transaction)}`)
+    navigation.navigate(ScreenName.CeloWithdrawSelectDevice, {
+      ...route.params,
+      transaction,
+    });
+  };
+
+  if (!account || !transaction) return null;
+  const { pendingWithdrawals } = (account as CeloAccount).celoResources;
+  
+  if (pendingWithdrawals) {
+    if ((transaction.index === null || transaction.index === undefined) && pendingWithdrawals[0]) {
+      // TODO Verify what happens in the case there are pending withdrawals but nothing can be claimed (i.e. all are over 3 days)
+      onChange(pendingWithdrawals[0].index);
+    }
+  }
+
+  // const pendingWithdrawals = [
+  //   {
+  //     value: new BigNumber("6660000000000000"),
+  //     time: new BigNumber("1662042839"),
+  //     index: 0,
+  //   },
+  //   {
+  //     value: new BigNumber("1000000000000000"),
+  //     time: new BigNumber("1661439326"),
+  //     index: 1,
+  //   },
+  //   {
+  //     value: new BigNumber("6660000040000000"),
+  //     time: new BigNumber("1662042839"),
+  //     index: 2,
+  //   },
+  //   {
+  //     value: new BigNumber("1000000000000000"),
+  //     time: new BigNumber("1661439326"),
+  //     index: 3,
+  //   },
+  //   {
+  //     value: new BigNumber("6660000000000000"),
+  //     time: new BigNumber("1662042839"),
+  //     index: 4,
+  //   },
+  //   {
+  //     value: new BigNumber("1000000000000000"),
+  //     time: new BigNumber("1661439326"),
+  //     index: 5,
+  //   }
+  // ];
+
+
+  const [infoModalOpen, setInfoModalOpen] = useState(false);
+
+  const openModal = useCallback(time => setInfoModalOpen(time), [
+    setInfoModalOpen,
+  ]);
+  const closeModal = useCallback(() => setInfoModalOpen(false), [
+    setInfoModalOpen,
+  ]);
+  
+
+
+  const { amount } = status;
+  const unit = getAccountUnit(account);
+  const formattedAmount  = formatCurrencyUnit(unit, new BigNumber(amount), {
+    disableRounding: true,
+    alwaysShowSign: false,
+    showCode: true,
+  });
+
+  const formatAmount = (val) => {
+    return formatCurrencyUnit(unit, new BigNumber(val), {
+      disableRounding: true,
+      alwaysShowSign: false,
+      showCode: true,
+    });
+  }
+
+  // Note: if we want to mimick the same behavior as LLD, we'll need MomentJS to handle all date edge cases
+  const getDate = (time: string) => {
+    return new Date(parseInt(time) * 1000).toLocaleDateString();
+  }
+
+  const hasErrors = Object.keys(status.errors).length > 0;
+
+  return (
+      <SafeAreaView style={[styles.root, { backgroundColor: colors.background }]}>
+        <TrackScreen category="WithdrawFlow" name="Amount" />
```

Comment:
Nit: Maybe `CeloWithdraw` to be consistent with the other screens?

Reply:
fixed